.circleci/config.yml

```diff
@@ -126,6 +126,26 @@ references:
           owner=${CIRCLE_PROJECT_USERNAME} repo=${CIRCLE_PROJECT_REPONAME} \
           tag=${CIRCLE_TAG} ASSETS=${PACKAGES_DIR:?}/*
 
+  fail_notification: &fail_notification
+    run:
+      name: Send failure notification
+      environment:
+        HOOK_TEMPLATE: |
+          {
+            "text": "CircleCI job **%s** failed on branch **%s**",
+            "attachments": [
+              {
+                "title": "Build Link",
+                "title_link": "%s",
+                "color": "#FAD6D6"
+              }
+            ]
+          }
+      command: |
+        HOOK_DATA=$(printf "$HOOK_TEMPLATE" "${CIRCLE_JOB:?}" "${CIRCLE_BRANCH:?}" "${CIRCLE_BUILD_URL:?}")
+        curl -X POST -H 'Content-Type: application/json' ${ROCKET_HOOK_URL:?} --data "${HOOK_DATA:?}"
+      when: on_fail
```

Comment:
I see - if any one of previous steps returned non-zero exit status:
https://circleci.com/docs/2.0/configuration-reference/#run
https://circleci.com/docs/2.0/configuration-reference/#the-when-attribute